pkg/trafficpolicy/trafficpolicy.go

```diff
@@ -33,6 +33,16 @@ func NewOutboundTrafficPolicy(name string, hostnames []string) *OutboundTrafficP
 	}
 }
 
+// TotalClustersWeight returns total weight of the WeightedClusters in trafficpolicy.RouteWeightedClusters
+func (rwc *RouteWeightedClusters) TotalClustersWeight() int {
```

Comment:
A unit test for this method too would be great

Reply:
will add one now. thanks